### DISCUSSION TITLE
Meta: Stale Bot response time

### DISCUSSION THREAD

**Discussion by sylvank** [snipe/snipe-it#10252] [General]
### Is your feature request related to a problem? Please describe. A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

This is a comment on the management of the Snipe-It GitHub Issues:

I am a new Snipe-It user, and when googling problems or questions I have, the Github Issues section is the most commonly referenced resource. 

On almost every post or discussion I've looked into, there are multiple comments by Stale Bot asking for updates, then closing the Issue, followed by users commenting again. It looks like it closes Issues after 2 months of inactivity. 

Just one example: https://github.com/snipe/snipe-it/issues/5425

### Describe the solution you'd like A clear and concise description of what you want to happen.

Based on the amount and frequency of activity of Snipe-It users, I would like to suggest a delay of at least 1 year before Stale Bot attempts to close an issue as being more appropriate, if this action is even considered useful at all. See below for consideration:

https://drewdevault.com/2021/10/26/stalebot.html

Thank-you.

**Comment by welcome** [comment]
👋 Thanks for opening your first issue here! If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
